### PR TITLE
PA 1.3.1

### PR DESCRIPTION
**Pull request by MariekeGO**

### CODE REVIEW

File: release-notes/PA/PA_1.3.1.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+> This version requires that **SRM 1.2.28 [CU1]** is installed.
+
+## New features
+
+#### Assigning tasks to a specific user or group [ID_35082]
+
+While up to now Process Automation created user tasks without assigning them to specific users, now you can configure a process definition so that tasks are assigned or reassigned to a user or a group of users when they are created.
+
+During the configuration of a process definition,you can select a Process DOM field for either the group or the user so that the group or user name can be retrieved when the user task gets created. With the *Fixed* option you can already pre-select the exact group and user. However, if a Process DOM field is selected for a group, you cannot pre-select a fixed user. The *Any* option allows you to postpone the selection of the group or user to the time when the task gets created.
+
```
Comment (edalslc) [inline release-notes/PA/PA_1.3.1.md:17]: Can you add a warning stating that this is first step towards being able to assign user tasks to specific users/groups? This is for now the configuration part; nothing will be done with the user/user group info.

**Review by edalslc — CHANGES_REQUESTED** [review]

File: release-notes/PA/PA_1.3.1.md (hunk 1 above)
Comment (MariekeGO) [reply release-notes/PA/PA_1.3.1.md]: Done!

File: release-notes/PA/PA_1.3.1.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+### Fixes
+
+#### No tokens processed if more tokens were in queue than the Concurrent Tasks parameter value [ID_35437]
+
+If a queue had more tokens with the *InQueue* status than the number indicated by the *Concurrent Tasks* parameter, it could occur that no more tokens were processed.
+
+The logic of the *Skyline Queue Manage* connector has now been adjusted so that the queue will be able to process multiple tokens at once, instead of processing them one by one like before.
+
+With this fix, another possible issue has been resolved, where if a pool resource was used and no regular resource was available to swap it with, the first token using the pool resource was not flagged as in error.
```
Comment (edalslc) [inline release-notes/PA/PA_1.3.1.md:111]: can we remove that last sentence (with this fix ... ) ?  This is related to a feature we don't want to promote

File: release-notes/PA/PA_1.3.1.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+The *Skyline Queue Manager* connector has been adjusted so that it no longer sends a message to itself when forwarding tokens, and it adds the token directly to the database. This will improve the performance of the connector.
+
+### Fixes
+
+#### No tokens processed if more tokens were in queue than the Concurrent Tasks parameter value [ID_35437]
+
+If a queue had more tokens with the *InQueue* status than the number indicated by the *Concurrent Tasks* parameter, it could occur that no more tokens were processed.
+
+The logic of the *Skyline Queue Manage* connector has now been adjusted so that the queue will be able to process multiple tokens at once, instead of processing them one by one like before.
```
Comment (edalslc) [inline release-notes/PA/PA_1.3.1.md:109]: Skyline Queue ManageR

**Review by edalslc — CHANGES_REQUESTED** [review]
two small items I've missed initially. Then good for me , can be merged.

**Pull request merged**